test/integration/an_item_can_be_removed_from_cart_test.rb

```diff
@@ -45,7 +45,7 @@ class AnItemCanBeRemovedFromCartTest < ActionDispatch::IntegrationTest
 
     click_link 'Remove Item'
     within('.flash_remove') do
-      assert page.has_content?('You have removed the item gnar possum from your cart.')
+      assert page.has_content?('You have removed the item: (gnar possum) from your cart.')
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.